cmd/lotus-shed/chain_index.go

```diff
@@ -0,0 +1,191 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/urfave/cli/v2"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-state-types/abi"
+
+	lcli "github.com/filecoin-project/lotus/cli"
+)
+
+var chainIndexCmds = &cli.Command{
+	Name:            "chainindex",
+	Usage:           "Commands related to managing the chainindex",
+	HideHelpCommand: true,
+	Subcommands: []*cli.Command{
+		withCategory("chainindex", validateBackfillChainIndexCmd),
+	},
+}
+
+var validateBackfillChainIndexCmd = &cli.Command{
+	Name:  "validate-backfill",
+	Usage: "Validates and optionally backfills the chainindex for a range of epochs",
+	Description: `
+lotus-shed chainindex validate-backfill --from <start_epoch> --to <end_epoch> [--backfill] [--log-good]
```

Comment:
:+1: to `--to`, every time I have to `--epochs` I have to calculate how many based on the actual epoch I want to go to!